internal/local/zip.go

```diff
@@ -27,8 +27,8 @@ type ZipDB struct {
 	Name string
 	// the url that the zip archive was downloaded from
 	ArchiveURL string
-	// whether this database should make any network requests
-	Offline bool
```

Comment:
fwiw I think it would be better to keep the existing structure within the database logical, as it's technically possible they could be implemented to make network requests before downloading and (more practically) it means a smaller diff + aligns better with everything else (e.g. the error is `ErrOfflineDatabaseNotFound`, the tests say "if the db is offline", etc).

The fact that we're exposing this to the user as "download databases" is something of an implementation detail of the CLI.

Having said that, I won't block if you can't be bothered reverting 🙂

Reply:
I just feel it's weird to use `downloadDB` to indicate `offline` so would like to keep consistent in the names, but I don't mind too much keeping `offline` in `ZipDB`.